luigi/contrib/hive.py

```diff
@@ -180,8 +180,12 @@ class MetastoreClient(HiveClient):
     def table_location(self, table, database='default', partition=None):
         with HiveThriftContext() as client:
             if partition is not None:
-                partition_str = self.partition_spec(partition)
-                thrift_table = client.get_partition_by_name(database, table, partition_str)
+                try:
+                    import hive_metastore.ttypes
+                    partition_str = self.partition_spec(partition)
+                    thrift_table = client.get_partition_by_name(database, table, partition_str)
+                except hive_metastore.ttypes.NoSuchObjectException as e:
```

Comment:
can you fix this flake8 issue?
`./luigi/contrib/hive.py:187:71: F841 local variable 'e' is assigned to but never used`

(btw you can run `tox -e flake8` locally to test)